lib/api_me.rb

```diff
@@ -14,6 +14,7 @@
 module ApiMe
   extend ActiveSupport::Concern
   include ::Pundit
+  include ActionController::Live
```

Comment:
Recommend putting this inside of the active support concern callback of `included`, see https://api.rubyonrails.org/classes/ActiveSupport/Concern.html for a better description than I could write.